src/main/webapp/app/exercises/programming/manage/build-plan-editor.scss

```diff
@@ -2,12 +2,15 @@
     flex: 1 1 auto;
     display: flex;
     &__content {
+        height: 60vh;
```

Comment:
Is there a way with the new editor to just let it always use the full remaining height of the page? This would remove the unused space below the editor.
The fixed height for the old editor was only there because resizing it behaved a bit weird sometimes.

Reply:
I can have a look 👍